util/dvsim/dvsim.py

```diff
@@ -401,6 +401,13 @@ def main():
         metavar="N",
         help="""Interval in seconds. Print status every N seconds.""")
 
+    parser.add_argument(
```

Comment:
I dont think there is a need to control this via cmd line switch. We can just set the default to 20.